### PR TITLE
Add testcase for issue 850

### PR DESCRIPTION
**Pull request by DelusionalLogic**
I hope this is how you want the testcase to be laid out.

Testcase for #850

### CODE REVIEW

File: test/issues/850/vimrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set nocompatible
+let &rtp  = '~/.vim/bundle/FastFold,' . &rtp
+let &rtp  = '~/.vim/bundle/vimtex,' . &rtp
+let &rtp .= ',~/.vim/bundle/vimtex/after'
+filetype plugin indent on
+syntax enable
+
+" Here we need to edit a large tex file, e.g. a thesis or long paper
+edit main.tex
```
Comment (lervag) [inline test/issues/850/vimrc]: If you prepend `silent` we don't need to type `<cr>` when starting the test.

File: test/issues/850/vimrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+set nocompatible
+let &rtp  = '~/.vim/bundle/FastFold,' . &rtp
```
Comment (lervag) [inline test/issues/850/vimrc]: Do we really need fastfold here?

File: test/issues/850/vimrc [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+let &rtp  = '~/.vim/bundle/vimtex,' . &rtp
+let &rtp .= ',~/.vim/bundle/vimtex/after'
+filetype plugin indent on
+syntax enable
+
```
Comment (lervag) [inline test/issues/850/vimrc]: I like to map `q` to `:qall` in order to make it easy to exit the test. See e.g. the template for how I prefer to do this.

**Comment by DelusionalLogic** [comment]
I tried to make it as low friction as possible by opening the TOC automatically as well.

I can't figure out how to make Vimtex not hijack q on TOC open, so you'll have to press q twice to exit. I hope that's ok, Otherwise you'll have to point me in the direction of how to do it properly.

**Comment by lervag** [comment]
This is good, thanks. :)

**Pull request merged**